src/test/java/seedu/address/logic/commands/classcommands/RemoveFromClassCommandTest.java

```diff
@@ -0,0 +1,326 @@
+package seedu.address.logic.commands.classcommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.classcommands.RemoveFromClassCommand.MESSAGE_REMOVE_SUCCESS;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TestUtil.getIndexList;
+import static seedu.address.testutil.TypicalTimestable.ALICE;
+import static seedu.address.testutil.TypicalTimestable.CARL;
+import static seedu.address.testutil.TypicalTimestable.getTypicalStudents;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Student;
+import seedu.address.model.person.UniqueStudentList;
+import seedu.address.model.tuitionclass.ClassName;
+import seedu.address.model.tuitionclass.ClassTiming;
+import seedu.address.model.tuitionclass.Location;
+import seedu.address.model.tuitionclass.Rate;
+import seedu.address.model.tuitionclass.StudentNameList;
+import seedu.address.model.tuitionclass.TuitionClass;
+import seedu.address.model.tuitionclass.UniqueClassList;
+
+public class RemoveFromClassCommandTest {
+
+
+    @Test
+    public void constructor_nullInput_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new RemoveFromClassCommand(null));
+    }
+
+    @Test
+    public void execute_removeSingleStudent_success() {
+        // model stub with a class containing ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE.
+        Model model = new ModelStubWithClass(getTypicalStudents());
```

Comment:
Do u want to rename it ModelStubWithClass modelStub = ...
similar to what they do in AddCommandTest?